src/sql/workbench/contrib/dataExplorer/browser/media/connectionViewletPanel.css

```diff
@@ -137,14 +137,28 @@
 .monaco-tree .monaco-tree-rows > .monaco-tree-row > .content > .connection-tile > .icon.server-page,
 .hc-light .monaco-tree .monaco-tree-rows > .monaco-tree-row > .content > .connection-tile > .icon.server-page,
 .monaco-list .monaco-list-rows > .monaco-list-row .connection-tile > .icon.server-page,
-.hc-light .monaco-list .monaco-list-rows > .monaco-list-row .connection-tile > .icon.server-page {
+.hc-light .monaco-list .monaco-list-rows > .monaco-list-row .connection-tile > .icon.server-page,
+.monaco-tree .monaco-tree-rows > .monaco-tree-row > .content > .connection-profile-container > .icon.server-page,
+.hc-light .monaco-tree .monaco-tree-rows > .monaco-tree-row > .content > .connection-profile-container > .icon.server-page,
+.monaco-list .monaco-list-rows > .monaco-list-row .connection-profile-container > .icon.server-page,
+.hc-light .monaco-list .monaco-list-rows > .monaco-list-row .connection-profile-container > .icon.server-page {
 	background: url('default_server.svg') center center no-repeat;
 }
 
+.monaco-list:focus .monaco-list-rows > .monaco-list-row.selected .connection-profile-container > .icon.server-page,
+.hc-light .monaco-list:focus .monaco-list-rows > .monaco-list-row.selected .connection-profile-container > .icon.server-page {
+	background: url('default_server_inverse.svg') center center no-repeat !important;
```

Comment:
Why is the `!important` needed? Is there some property being set directly on the element itself that overrides this?

Generally we should avoid doing overrides like this since it can conflict with other style changes that come in later on.